opencti-platform/opencti-front/src/private/components/data/connectors/Connector.jsx

```diff
@@ -281,15 +285,42 @@ const ConnectorComponent = ({ connector, relay }) => {
         <div className={classes.popover}>
           <Security needs={[MODULES_MODMANAGE]}>
             <Tooltip title={t_i18n('Reset the connector state')}>
-              <IconButton
-                onClick={handleOpenResetState}
-                aria-haspopup="true"
-                color="primary"
-                size="large"
-                disabled={connector.built_in}
-              >
-                <PlaylistRemoveOutlined/>
-              </IconButton>
+              {isSensitive ? (
+                <div style={{ position: 'relative', display: 'inline-block' }}>
+                  <DangerZoneBlock
+                    type="connector_reset"
+                    sx={{
+                      root: { border: 'none', padding: 0, margin: 0 },
+                      title: { position: 'absolute', zIndex: 2, left: 4, top: 9, fontSize: 8 },
+                    }}
+                  >
+                    {({ disabled }) => (
+                      <Button
```

Comment:
![image](https://github.com/user-attachments/assets/f21a5fe5-5dd3-4b7a-a593-31e757e19701)

A little croped in the pc screen